Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ruby:alpine
```

Comment:
In my experience, it is not the best way to use such a generic image without any version information. Maybe it is better to use a more concrete image tag.